src/github_repo_request_local.py

```diff
@@ -0,0 +1,74 @@
+import subprocess
```

Comment:
Generally I like us to include some file level documentation that briefly explains why this code exists.

Please can you add a docstring """ .... """ to describe this file's purpose?

Reply:
`Docstring` has been added and committed.